ansible/undercloud/templates/introspect.sh.j2

```diff
@@ -19,7 +19,8 @@ source /home/stack/stackrc
 ## * Introspect hardware attributes of nodes.
 ## ::
 
-openstack baremetal introspection bulk start
+openstack overcloud node introspect --all-manageable --provide
```

Comment:
Should we check version and use the old bulk way for version < 13?

Reply:
I checked for OSP10 and this command works...so I think we are good.